src/UserService.Business/Commands/User/CreateUserCommand.cs

```diff
@@ -314,12 +322,17 @@ public OperationResultResponse<Guid> Execute(CreateUserRequest request)
 
             SendEmail(dbUser, password, response.Errors);
 
-            if (request.DepartmentId.HasValue)
+            if (request.DepartmentId.HasValue && !ChangeUserDepartment(request.DepartmentId.Value, userId, response.Errors))
             {
```

Comment:
Department i not required, it partial not failed

Reply:
Fixed